system-tests/lib/system-tests.ts

```diff
@@ -265,6 +269,7 @@ export type SpawnerResult = {
   stderr: stream.Readable
   on(event: 'error', cb: (err: Error) => void): void
   on(event: 'exit', cb: (exitCode: number) => void): void
+  kill?: ChildProcess['kill']
```

Comment:
nit: i'm guessing this is optional here because `DockerProcess#kill` doesn't exist, and without this, type checking fails. i get it if you don't want to implement `DockerProcess#kill`, but IMO this would be slightly cleaner if you added a stub `kill` method to alert the dev the reason `DockerProcess#kill` isn't working. something like this:

```
kill() {
  throw new Error('.kill not implemented for DockerProcess.')
}
```